Galway 2020/css/custom.css

```diff
@@ -154,84 +154,79 @@ body {
 /* ************************************************************************** */
 
 /*price range search*/
-.filterLi {
-  float: left;
-  width: 20%;
-  line-height: 20px;
-  text-align: center;
-  margin: 5px;
-  display: none; 
-  background: #f2f4f7;
-  box-shadow: 10px 10px 10px 0 #eff1f8;
-  border: 1px solid #ebeced;
-  border-radius: 10px 10px 10px 10px;
-}
-.list{
-	text-decoration: none;
-	color: #666;
-	font-size: 0.9em;
-	text-transform: uppercase;
-	text-shadow: 2px 2px 5px #666;
-	letter-spacing: 0.4em;
-	
-}
-.list:hover{
-	text-decoration: none;
-	color: #666;
-	font-size: 1em;
-	text-transform: uppercase;
-}
-.pricetag{
-	font-size: 0.9em;
-	text-decoration: none;
-	color: #666;
-	text-transform: uppercase;
-}
-.smallsp{
-	font-size: 0.6em;
-}
-hr{
-	width: 40%;
-	border: 1px solid orange;
-}	
-.show {
-  display: block;
-}
-
-.container {
-  margin-top: 20px;
-  overflow: hidden;
+ .filterLi {
+     float: left;
+     width: 20%;
+     line-height: 20px;
+     text-align: center;
+     margin: 5px;
+     display: none;
+     background: #f2f4f7;
+     box-shadow: 10px 10px 10px 0 #eff1f8;
+     border: 1px solid #ebeced;
+     border-radius: 10px 10px 10px 10px;
+}
+ .list{
+     text-decoration: none;
+     color: #666;
+     font-size: 0.9em;
+     text-transform: uppercase;
+     text-shadow: 2px 2px 5px #666;
+     letter-spacing: 0.4em;
+}
+ .list:hover{
+     text-decoration: none;
+     color: #666;
+     font-size: 1em;
+     text-transform: uppercase;
+}
+ .pricetag{
+     font-size: 0.9em;
+     text-decoration: none;
+     color: #666;
+     text-transform: uppercase;
+}
+ .smallsp{
+     font-size: 0.6em;
+}
+ hr{
+     width: 40%;
+     border: 1px solid orange;
+}
+ .show {
+     display: block;
+}
+ .container {
+     margin-top: 20px;
+     overflow: hidden;
 }

```

Comment:
Price filter is displaying perfectly as intended. It nicely floats on left with nice color choice as well.